source/settings/command-line-options.txt

```diff
@@ -138,23 +157,54 @@ value in the :guilabel:`Settings` panel.
 
        To learn more, see :ref:`compass-configure-network-traffic`.
 
+   * - .. option:: --password
+     - Specifies a password for authentication to Compass and the MongoDB 
+       deployment provided in the connection string. 
+
+       Use in conjunction with :option:`--username`. 
+
+   * - .. option:: --persistOIDCTokens
+     - Remain logged in when using the MongoDB OIDC authentication mechanism for 
+       MongoDB server connections. Session tokens are encrypted and stored using 
```

Comment:
[suggestion] another suggestion for consistency

```suggestion
   * - .. option:: --persistOIDCTokens
     - Allows you to remain logged in when using the MongoDB OIDC authentication mechanism for 
       MongoDB server connections. Session tokens are encrypted and stored using 
```

Reply:
Updated.